### PR TITLE
Multi-node kind cluster

### PR DESCRIPTION
**Pull request by katrybacka**

### CODE REVIEW

**Comment by katrybacka** [comment]
recheck ha

**Comment by katrybacka** [comment]
recheck ha

**Comment by katrybacka** [comment]
recheck ha

File: test/env/create_k8s_cluster.sh [hunk 1]
```diff
@@ -2,29 +2,50 @@
...
+networking:
+  disableDefaultCNI: true
+nodes:
 EOF
+)"
+
+node="
+- role: control-plane"
+
+for (( i=0; i<$number_of_nodes; i++ ))
```
Comment (kaweue) [inline test/env/create_k8s_cluster.sh:34]: I would use some yq here to add items to the array
`yq w sample.yaml "b.d[+]" "bar thing"`
However it is just a nit.

**Comment by katrybacka** [comment]
recheck

**Comment by katrybacka** [comment]
recheck ha

**Comment by katrybacka** [comment]
recheck

**Pull request merged**